lib/core/engine/collector.js

```diff
@@ -357,6 +357,14 @@ class Collector {
         });
       }
 
+      // Add Firefox perfStats if available
+      if (this.options.firefox && this.options.firefox.perfStats) {
+        if (!results.perfStats) {
+          results.perfStats = [];
```

Comment:
Hi @dpalmeiro sorry I've been offline for a week, I'm back now. I haven't tested the perfStats yet, but what about just rename the field _perfStats_ to something more descriptive? Is it internal browser performance statistics or what is collected?

Reply:
Sure, I'm open to any naming changes.  Maybe geckoPerfStats or something if that is more clear, or anything you may think sounds better.

Reply:
Yes, geckoPerfStats sounds get better, I'm just thinking so it do not collide with something else in the future. I can change it if you want, I wanna push the release later tonight.

Reply:
Sure, please go ahead. Thanks!